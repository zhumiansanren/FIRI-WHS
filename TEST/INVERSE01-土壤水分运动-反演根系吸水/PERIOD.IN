!==============================================================================
! 应力期数据
!
! PID    -- 应力期序号
! PERLEN -- 每个应力期的应力期长度, d
! NSTP   -- 每个应力期的时段数量
! TSMLT  -- 每个应力期的步长增量因子
! OTOP   -- 每个应力期的地表含水率, cm^3 cm^-3
!==============================================================================
  PID      PERLEN      NSTP      TSMLT      OTOP
    1         1.0       100       1.0        0.3
