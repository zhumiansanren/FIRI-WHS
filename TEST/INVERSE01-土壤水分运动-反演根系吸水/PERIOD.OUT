================================================================================
                              农田水热盐运移数值模型
                                    FIRI-WHS
                            中国农业科学院农田灌溉研究所
================================================================================

项目名称: This is a demo of inversing root water uptake.
应力期数量:            1

      PERIOD      PERLEN        TIME     STORAGE        WBOT      C_WBOT        WTOP      C_WTOP
           1      1.0000      1.0000     20.1162  -4.162E-03  -4.162E-03   1.120E+00   1.120E+00
